src/coreclr/jit/phase.cpp

```diff
@@ -221,7 +223,7 @@ void Phase::PostPhase(PhaseStatus status)
         // cse
         // assertion prop
         // range check
-        // update flow
+        // PHASE_OPT_UPDATE_FLOW_GRAPH,
```

Comment:
Do you want to un-comment this?

Reply:
I had it that way at one point, but it caused loop table validity asserts. 

Will revisit as part of round 2 when I try converting the rest of the phases.